### ISSUE TITLE
Rename repository to cg-dashboard

### ISSUE THREAD

**Issue by msecret** [18F/cg-dashboard#412]
In order for users to consistently see us calling the dashboard the correct name, all user-facing, different names such as "console" or "deck" should be named to "dashboard".

**Comment by msecret** [comment]
I think this can ignore renaming things like repos, variables in code, etc for now. That can wait until the community release.

**Comment by brittag** [comment]
People might also consider converting any uppercase Cloud.gov to lowercase cloud.gov while working on this - that's a nice-to-have thing if convenient.

**Comment by msecret** [comment]
Agreed, we should definitely try and do that as well.

**Comment by msecret** [comment]
We could change this story to rename the repo to dashboard.

**Comment by jeremiak** [comment]
I renamed the issue to reflect ^^. This will involve not just a repo renaming, but also adjusting the Go code because `18f/cg-deck` is in the import path

**Comment by msecret** [comment]
@jcscottiii how do we coordinate this change with the least amount of breaking? Should we 1. update all the go code to point to `cg-dashboard` and commit a PR, then 2. change the repo name, and 3. see that the code passes 4. merge the PR?

**Comment by jcscottiii** [comment]
Before the repo name is changed, I imagine the following stuff must happen first:

Let's break this into two parts: 1) Deploying apps called `cg-dashboard` instead of `cf-deck` in cloud foundry and 2) Fixing the Go stuff because doing it all at one time will be confusing.

## Deploying apps called `cg-dashboard`

1. Create a PR where all the deployment references in the manifests and in `travis_deploy.sh` use `cg-dashboard` instead of `cf-deck`. (don't merge)
1. At this point, you are freezing deployments, go to all the spaces in both api.cloud.gov and api.fr.cloud.gov (eastwest and govcloud respectively) and rename all the `cf-deck` apps to `cg-dashboard`. This will allow zero-downtime-deploy to just redeploy on the new app names.
1. Quickly merge that PR lol.


## Deploy Go code under the new package name

1. Create a PR where the Go package name uses the new name in the code and the imports that import neighboring packages. (don't merge).
  - [ ] Also, change the package name in https://github.com/18F/cg-deck/blob/master/glide.yaml
  - [ ] Also, change to `cg-dashboard` in Procfile
  - [ ] Also, change to `cg-dashboard` in .gitignore
1. **Finally rename the repo to `cg-dashboard`**
1. Merge it :shipit:

**Comment by msecret** [comment]
I'm a little blocked on this because I think the change should happen to Jame's PR to switch the deploy to circle. But I can't put this work on that work because I don't have access to his fork.

**Comment by jcscottiii** [comment]
There are some frontend things that have `cg-deck` still.
As well as the README.

@msecret

**Comment by jeremiak** [comment]
I didn't see anything in the README, but I made some changes to the front end and `env.sample` in #505

**Comment by jeremiak** [comment]
This is surely done, nice work team!!
